jupyterlab/labbox_ephys_widgets_jp/src/extensions/timeseries/TimeseriesViewNew/TimeseriesViewNew.tsx

```diff
@@ -71,7 +71,7 @@ const TimeseriesViewNew = (props: Props) => {
                     const slot = await timeseriesCalculationPool.requestSlot();
                     try {
                         result = await hither.createHitherJob(
-                            'get_timeseries_segment',
+                            'createjob_get_timeseries_segment',
```

Comment:
The jupyter extension only supports the new kind of hither calls where we have a createjob function that in turn creates and returns a hither job.